docs/source/markdown/podman-build.1.md

```diff
@@ -784,21 +878,29 @@ mechanism:
 
 Exclude all doc files except Help.doc from the image.
 
-This functionality is compatible with the handling of .dockerignore files described here:
+This functionality is compatible with the handling of .dockerignore files
+described here:
 
 https://docs.docker.com/engine/reference/builder/#dockerignore-file
 
 **registries.conf** (`/etc/containers/registries.conf`)
 
-registries.conf is the configuration file which specifies which container registries should be consulted when completing image names which do not include a registry or domain portion.
+registries.conf is the configuration file which specifies which container
+registries should be consulted when completing image names which do not include
+a registry or domain portion.
 
 ## Troubleshooting
 
 ### lastlog sparse file
 
-If you are using a useradd command within a Containerfile with a large UID/GID, it will create a large sparse file `/var/log/lastlog`.  This can cause the build to hang forever.  Go language does not support sparse files correctly, which can lead to some huge files being created in your container image.
+If you are using a useradd command within a Containerfile with a large UID/GID,
+it will create a large sparse file `/var/log/lastlog`.  This can cause the
+build to hang forever.  Go language does not support sparse files correctly,
+which can lead to some huge files being created in your container image.
 
-If you are using `useradd` within your build script, you should pass the `--no-log-init or -l` option to the `useradd` command.  This option tells useradd to stop creating the lastlog file.
+If you are using `useradd` within your build script, you should pass the
+`--no-log-init or -l` option to the `useradd` command.  This option tells
+useradd to stop creating the lastlog file.
```

Comment:
This is also in the troubleshooting guide.  I think this makes sense to leave as is here, but maybe we should add a pointer to the troubleshooting guide here too?